docs/porting/target/target.md

```diff
@@ -85,3 +85,18 @@ The Mbed OS HAL provides a set of conformance tests for the Mbed OS HAL APIs. Yo
 ```
 mbed test -t <toolchain> -m <target> -n mbed-os-tests-mbed_hal*
 ```
+
```

Comment:
if PR https://github.com/ARMmbed/mbed-os/pull/10820 will come in in current form, then:
`mbed-os-tests-mbed_hal*`  -> `tests-mbed_hal_fpga_ci_test_shield*`.

Reply:
I just realized that this line is for general HAL testing, so please ignore this comment.